packages/shared/package.json

```diff
@@ -19,6 +19,7 @@
     "dependencies": {
         "@aws-sdk/client-s3": "3.348.0",
         "@datadog/datadog-api-client": "^1.16.0",
+        "@google-cloud/bigquery": "^7.5.1",
```

Comment:
can we pin dependency? every google sdk are auto generated and they are know to introduce small breaking changes